setup.py

```diff
@@ -34,7 +34,7 @@
     "Snowflake, BigQuery, Redshift, data reliability, analytics engineering",
     long_description=README,
     install_requires=[
-        "click>=7.0,<9",
+        "click>=7.0,<8.1.4",
```

Comment:
There are typing bugs that were introduced specifically in click 8.1.4,
there's an active issue about it:
https://github.com/pallets/click/issues/2558

Reply:
Once they release a fix we can return the upper bound to be as it was before